swagger.yaml

```diff
@@ -4630,3 +4630,35 @@ definitions:
         type: string
         enum: [beta, production]
         description: Level of support of the version.
+
+  operator-property:
+    type: object
+    properties:
+      name:
+        type: string
+        description: Name of the property
+      type:
+        type: string
+        enum: ['boolean', 'string', 'number']
+        description: Type of the property
```

Comment:
Do we really need it? Shouldn't string would be the most simple? Otherwise, it could be extended to float, and arrays, etc.

Reply:
The idea here was to give the UI some clues on how to validate the user input.